content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md

```diff
@@ -52,3 +52,167 @@ bash-completion sources all completion scripts in `/etc/bash_completion.d`.
 {{< /note >}}
 
 Both approaches are equivalent. After reloading your shell, kubectl autocompletion should be working.
+
+### Completion combined with aliases inspired by [oh-my-zsh](https://github.com/ohmyzsh/ohmyzsh) plugin kubectl
```

Comment:
I like the aliases, yet I'm not sure if it is the right place (auto-completion).
The setup of auto-completion is a useful not obvious step, while the aliases are nice to do if relevant for someone